server-native.yaml

```diff
@@ -0,0 +1,81 @@
+- name: native-builder
+  version: 1.0
+  from: quay.io/quarkus/centos-quarkus-maven:20.0.0-java11
+  description: Builder image for native components
+  artifacts:
+  - name: config-generator-src
+    url: https://github.com/infinispan/infinispan-image-artifacts/archive/2.0.0.Dev01.tar.gz
+    target: config-generator-src.tar.gz
+  - name: server-src
+    url: https://github.com/infinispan/infinispan-quarkus/archive/11.0.0.Dev05.tar.gz
+    target: server-src.tar.gz
+  modules:
+    repositories:
+    - path: modules
+    install:
+    - name: quarkus.config-generator
+    - name: quarkus.server
+
+- name: infinispan/server-native
+  version: 11.0.0-SNAPSHOT
+  description: Infinispan Quarkus Server
+  from: registry.access.redhat.com/ubi8/ubi-minimal:8.2-267
+  artifacts:
+  - name: config-generator
+    image: native-builder
+    path: /opt/config-generator
+  - name: server
+    image: native-builder
+    path: /opt/server
+  packages:
+    manager: microdnf
+  ports:
+  - value: 2157
+  - value: 7800
+  - value: 11221
+  - value: 11222
+  - value: 45700
+  - value: 57600
+  labels:
+  - name: name
+    value: Infinispan Server
+  - name: version
+    value: 11.0.0.Dev04
```

Comment:
Shouldn't this be Dev05 ?

Reply:
Yep just copy and pasted from the old `image.yaml`, I'll fix.